rb/lib/selenium/webdriver/chrome/bridge.rb

```diff
@@ -39,11 +37,11 @@ def initialize(opts = {})
           caps = create_capabilities(opts)
 
           remote_opts = {
-            :url                  => url,
-            :desired_capabilities => caps
+            url: url,
```

Comment:
Personally, I prefer it, but if we can find the rubocop setting to enforce it, I'm ok to keep the longer indents.